### PR TITLE
Rollup

### PR DESCRIPTION
**Pull request by JakeHedman**

### CODE REVIEW

**Comment by JakeHedman** [comment]
ping

File: reactfire/rollup.config.js [hunk 1]
```diff
@@ -1,21 +1,35 @@
...
-  ],
-  plugins: [resolve()]
+  ]
 };
+
+export default [
+  {
+    ...config,
+    output: {
+      dir: './pub/reactfire/cjs',
```
Comment (iwikal) [inline reactfire/rollup.config.js:22]: What's the `reactfire` subdirectory for? I've seen others do it, but I don't understand why.

Comment (iwikal) [reply reactfire/rollup.config.js:22]: I see now that this isn't something you added.

**Review by iwikal — APPROVED** [review]
Thanks an alot!

**Pull request merged**